app/services/idv/steps/inherited_proofing/verify_wait_step.rb

```diff
@@ -0,0 +1,13 @@
+module Idv
+  module Steps
+    module InheritedProofing
+      class VerifyWaitStep < InheritedProofingBaseStep
+        include UserPiiManagable
```

Comment:
I think this can be removed.

Reply:
You're right. Good catch. I missed a bunch of these in clean up